website/docs/how-to/how-to-setup-provisioning-with-entra.md

```diff
@@ -81,17 +83,11 @@ You should remove all unnecessary properties. This ensures that Entra will reach
 - emails
 - externalId
 
-**4) Update the active property**
-
-You need to set the active property to the following (lowercase the boolean values):
-
-Switch([IsSoftDeleted], , "false", "true", "true", "false")
-
-**5) Update the email property to **
+**4) Update the email property to "userPrincipleName"**
 
 ![Update provisioning properties](/img/scim-entra-config-6.png)
 
-**6) Save**
+**5) Save**
```

Comment:
I think these steps do not align with the images, but let's fix this in another PR. I'll create an issue in the docs backlog

Reply:
Yeah... gosh... a really smart person told me once that images in docs were prone to this kind of thing :cry: